src/locators/customLocator.js

```diff
@@ -0,0 +1,37 @@
+var ControlLocator = require('./controlLocator');
```

Comment:
At the end, this file should go into a separate PR that Georgios will use as a sample. Or maybe move as commented section in the byControl ?

Reply:
moving it to a new pr